assembly/assembly-wsmaster-war/src/main/webapp/WEB-INF/classes/codenvy/che.properties

```diff
@@ -297,6 +301,9 @@ che.openshift.workspaces.pvc.quantity=10Gi
 che.openshift.workspace.cpu.limit=1
 # Override memory limit used for openshift workspaces. String, e.g. 1300Mi
 che.openshift.workspace.memory.override=NULL
+# Create secure route against HTTPS 
```

Comment:
@ibuziuk shouldn't we use just one property? I think that having 2 properties that actually mean the same thing (i.e. enable ssl) doesn't makes sense.